packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -512,13 +513,15 @@ const processCompareModels = async () => {
 		`;
 
 	const taskRes = await compareModels(modelIds.value, request, props.node.workflowId, props.node.id);
-	compareModelsTaskId = taskRes.id;
 	if (taskRes.status === TaskStatus.Success) {
+		compareModelsTaskId = taskRes.id;
 		generateOverview(taskRes.output);
+
+		const state = cloneDeep(props.node.state);
+		state.hasRun = true;
```

Comment:
I am not sure what `hasRun` and `isProcessingComparison` are doing exactly as I am not familiar with this but im surprised to see that `hasRun` is only `true` when we have `succeeded` (what about failing, cancelling ect)